File: VB/WinWebSolution.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Imports DevExpress.Persistent.BaseImpl

Namespace WinWebSolution.Module
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal session As Session, ByVal currentDBVersion As Version)
			MyBase.New(session, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Dim obj As TestAggregatedPropertiesObject = Session.FindObject(Of TestAggregatedPropertiesObject)(CriteriaOperator.Parse("Name=?", "TestAggregatedPropertiesObject"))
			If obj Is Nothing Then
				obj = New TestAggregatedPropertiesObject(Session)
				obj.Name = "TestAggregatedPropertiesObject"
				obj.Save()
			End If
		End Sub
	End Class
End Namespace

File: CS/E1284/WinWebSolution.Module/Updater.cs
using System;

using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
using DevExpress.Persistent.BaseImpl;

namespace WinWebSolution.Module {
    public class Updater : ModuleUpdater {
        public Updater(Session session, Version currentDBVersion) : base(session, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            TestAggregatedPropertiesObject obj = Session.FindObject<TestAggregatedPropertiesObject>(CriteriaOperator.Parse("Name=?", "TestAggregatedPropertiesObject"));
            if (obj == null) {
                obj = new TestAggregatedPropertiesObject(Session);
                obj.Name = "TestAggregatedPropertiesObject";
                obj.Save();
            }
        }
    }
}